src/webui/www/private/css/dynamicTable.css

```diff
@@ -52,6 +52,11 @@ tr.dynamicTableHeader {
     white-space: nowrap;
 }
 
+.dynamicTable tr.selected img,
+.dynamicTable tr:hover img {
+    filter: var(--color-icon-hover);
```

Comment:
This change breaks peer country flags when the line selected.

![screenshot](https://github.com/user-attachments/assets/6b0e606f-2eaf-411f-be90-9dd8892ed2ff)
![screenshot](https://github.com/user-attachments/assets/b3c166db-468b-4b6a-896a-31d12222007f)

Reply:
Will fix